src/Workspaces/Core/Portable/Workspace/Solution/SolutionState.cs

```diff
@@ -63,8 +62,7 @@ internal partial class SolutionState
             ImmutableDictionary<ProjectId, ProjectState> idToProjectStateMap,
             ImmutableDictionary<ProjectId, CompilationTracker> projectIdToTrackerMap,
             ImmutableDictionary<string, ImmutableArray<DocumentId>> filePathToDocumentIdsMap,
-            ProjectDependencyGraph dependencyGraph,
-            Lazy<VersionStamp> lazyLatestProjectVersion)
+            ProjectDependencyGraph dependencyGraph)
```

Comment:
any concern about partners using IVT to call these constructors?

Reply:
I would be very concerned if an IVT partner found some way to call this. :smile: